frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -75,6 +75,18 @@ class ChartSettings extends Component {
         return assocIn(this.props.series, [0, "card", "visualization_settings"], this.state.settings);
     }
 
+    getChartTypeName(){
+      switch(this.props.series[0].card.display){
+        case "table":
+          return "table";
+        case "scalar":
+          return "number";
+        default:
+          return "chart";
```

Comment:
I think we need to add an entry to our `.editorconfig` file for `.jsx` files. Right now we just have one for `.js` (@kdoh?). Your editor should be indenting things 4 spaces for you instead of 2 💔